project/LinkerdBuild.scala

```diff
@@ -12,10 +12,12 @@ import pl.project13.scala.sbt.JmhPlugin
  * - k8s/ -- finagle kubernetes client
  * - marathon/ -- marathon client
  * - router/ -- finagle router libraries
- * - linkerd/ -- configuration, runtime, and modules
+ * - namer/ -- name resolution
+ * - config/ -- configuration utilities
+ * - linkerd/ -- runtime, and modules
  * - test-util/ -- async test helpers; provided by [[Base]]
  */
-object LinkerdBuild extends Base {
+object LinkerdBuild extends Base { self =>
```

Comment:
Instead of `self`, the convention we've used previously is to just reference `LinkerdBuild` directly.  See lines 83, 91, 95, etc. for examples of that.  I think we should either switch everything in this file to use `self`, or drop this variable and use `LinkerdBuild` throughout.  I'm good with either approach.